inclusive-design/wecount.inclusivedesign.ca
# fix: Search overlaps header nav (resolves #264)

## Description
In iPad pro view the search in the header no longer overlaps the header navigation


## Steps to test

1. Go to deploy preview
2. Go to iPad Pro view
3. Observe header

**Expected behavior:** 
The search container should now be to the left of the header navigation links
## Additional information
May want to add more space between views link and search container


## Related issues


resolves #264
